### DISCUSSION TITLE
When are new docs being created?

### DISCUSSION THREAD

**Discussion by nelsonsvo** [Comcast/react-data-grid#2367] [Q&A]
Hi guys,

as you know your docs for canary are not here yet, just wondering when we should expect to see the new docs?

**Comment by BudgieInWA** [comment]
I am also interested in hearing from the maintainers on this.

There is mention of the new version on the "old" v5 website, but v7 is published to npm, so when you follow the [v5 Quickstart Guide](https://adazzle.github.io/react-data-grid/docs/quick-start) and run `yarn add react-data-grid` you are in for a very confusing ride. There is no guidance on which version is recommended, what features are lost when upgrading v5 to v7, and what the timeline or roadmap is for v7 and "canary" usually means "this is ready, but we are just making sure nothing is broken" not "we are still changing the API".

As you can see in #1752, this situation is leading to confusion and the appearance that the project is dead, when in fact there is active development and a sexy re-write around the corner. The README (which is the npm homepage) and the documentation site (for v5, which the README calls the "old site") need to make the situation clear, so all of this work is properly understood and recognised.

**Reply by pranaynailwal** [reply]
Exactly! I recently discovered this and found it really interesting based on the website overview. So I decided to implement it for my project. But now I am almost at that point of regretting that decision as I am not getting any clarity on what features to use and how to have them implemented. A task usually of minutes is taking hours. None of the examples or demos are helping as they are outdated. I have to read through the issues and repository to find the relevant implementations for my use cases. This is very tedious!
